mergify_cli/__init__.py

```diff
@@ -560,12 +570,14 @@ def get_default_token() -> str:
 
 def cli() -> None:
     global DEBUG
+    global DRAFT
```

Comment:
That should not be a global. (yeah the DEBUG way is clearly a bad practice)

Reply:
why not just pass the args.draft around where this is needed?

Reply:
I don't find that really handy to carry this param in every function that needed it even more if we refactor everything in the near (or not so far) future. And even if the global style is a bad practice, in this case, it's not really harmful it's not buried in N layers of dependencies.